### ISSUE TITLE
Issue with search_tesscut

### ISSUE THREAD

**Issue by rebekah9969** [KeplerGO/lightkurve#795]
I have been having issues when running search_tesscut as follows, 

```
target = 'KIC 8462852' 
search = lk.search_tesscut('KIC 8462852',  sector=None)
search
```

Which returns
```
KeyError                                  Traceback (most recent call last)
~/opt/anaconda3/envs/astroconda/lib/python3.6/site-packages/IPython/core/formatters.py in __call__(self, obj)
    700                 type_pprinters=self.type_printers,
    701                 deferred_pprinters=self.deferred_printers)
--> 702             printer.pretty(obj)
    703             printer.flush()
    704             return stream.getvalue()

~/opt/anaconda3/envs/astroconda/lib/python3.6/site-packages/IPython/lib/pretty.py in pretty(self, obj)
    392                         if cls is not object \
    393                                 and callable(cls.__dict__.get('__repr__')):
--> 394                             return _repr_pprint(obj, self, cycle)
    395 
    396             return _default_pprint(obj, self, cycle)

~/opt/anaconda3/envs/astroconda/lib/python3.6/site-packages/IPython/lib/pretty.py in _repr_pprint(obj, p, cycle)
    682     """A pprint that just redirects to the normal repr function."""
    683     # Find newlines and replace them with p.break_()
--> 684     output = repr(obj)
    685     lines = output.splitlines()
    686     with p.group():

~/Desktop/LIGHTKURVE/lightkurve/lightkurve/search.py in __repr__(self, html)
     70             return out
     71         columns = ['#', 'observation', 'author', 'target_name', 'productFilename', 'distance']
---> 72         return out + '\n\n' + '\n'.join(self.table[columns].pformat(max_width=300, html=html))
     73 
     74     def _repr_html_(self):

~/opt/anaconda3/envs/astroconda/lib/python3.6/site-packages/astropy/table/table.py in __getitem__(self, item)
   1653             return self.Row(self, item.item())
   1654         elif self._is_list_or_tuple_of_str(item):
-> 1655             out = self.__class__([self[x] for x in item],
   1656                                  copy_indices=self._copy_indices)
   1657             out._groups = groups.TableGroups(out, indices=self.groups._indices,

~/opt/anaconda3/envs/astroconda/lib/python3.6/site-packages/astropy/table/table.py in <listcomp>(.0)
   1653             return self.Row(self, item.item())
   1654         elif self._is_list_or_tuple_of_str(item):
-> 1655             out = self.__class__([self[x] for x in item],
   1656                                  copy_indices=self._copy_indices)
   1657             out._groups = groups.TableGroups(out, indices=self.groups._indices,

~/opt/anaconda3/envs/astroconda/lib/python3.6/site-packages/astropy/table/table.py in __getitem__(self, item)
   1647     def __getitem__(self, item):
   1648         if isinstance(item, str):
-> 1649             return self.columns[item]
   1650         elif isinstance(item, (int, np.integer)):
   1651             return self.Row(self, item)

~/opt/anaconda3/envs/astroconda/lib/python3.6/site-packages/astropy/table/table.py in __getitem__(self, item)
    237         """
    238         if isinstance(item, str):
--> 239             return OrderedDict.__getitem__(self, item)
    240         elif isinstance(item, (int, np.integer)):
    241             return self.values()[item]

KeyError: 'observation'
```

**Comment by barentsen** [comment]
Thanks so much for reporting this @rebekah9969!  This is a bug.  I believe it was freshly introduced when we merged #739 yesterday.

I plan to fix this bug as part of #796.

**Comment by barentsen** [comment]
This is now fixed on the master branch via PR #796.

As part of that PR, I added a regression test called `test_tesscut_795` to make sure this error won't re-appear.

Thanks again for reporting @rebekah9969! :+1:
